ignite/templates/field/fields.go

```diff
@@ -43,10 +43,17 @@ func (f Fields) ProtoImports() []string {
 	return allImports
 }
 
-// ProtoFieldName returns  all inline fields args for name used in proto.
-func (f Fields) ProtoFieldName() string {
+// ProtoFieldNameAutoCLI returns  all inline fields args for name used in proto.
+// It should be used in AutoCLI to generate the field name.
+func (f Fields) ProtoFieldNameAutoCLI() string {
 	args := ""
-	for _, field := range f {
+	for i, field := range f {
+		// only the last field can be a variadic field
+		if i == len(f)-1 && field.IsSlice() {
```

Comment:
Do we need to add the coins only as the last argument? Will the message not support a two-coin slice?

Reply:
may we should throw a warning or create a manual command in case the user tries to scaffold the slice of coins:

`ignite s list employee bonus:coins salary:coins`

Reply:
Creating a manual command for cases AutoCLI falls short would be a great feature yeah. I do not think we should have a warming however.
I can log an issue for that.